src/auto/injector.js

```diff
@@ -722,13 +722,14 @@ function createInjector(modulesToLoad, strictDi) {
       if (loadedModules.get(module)) return;
       loadedModules.put(module, true);
 
-      function runInvokeQueue(queue) {
+      function runInvokeQueue(queue, moduleFn) {
         var i, ii;
         for (i = 0, ii = queue.length; i < ii; i++) {
           var invokeArgs = queue[i],
               provider = providerInjector.get(invokeArgs[0]);
-
-          provider[invokeArgs[1]].apply(provider, invokeArgs[2]);
+          var args = Array.prototype.slice.call(invokeArgs[2]);
```

Comment:
Not so sure I like adding another arg... But can't think of another way to do it.